homeassistant/components/acmeda/cover.py

```diff
@@ -122,6 +123,6 @@ async def async_stop_cover_tilt(self, **kwargs: Any) -> None:
         """Stop the roller."""
         await self.roller.move_stop()
 
-    async def async_set_cover_tilt(self, **kwargs):
+    async def async_set_cover_tilt_position(self, **kwargs: Any) -> None:
```

Comment:
I spotted this whilst working on pylint plugin.
I'm pretty sure this was a bug, with the function name incorrect.

Reply:
Good spot! Thanks!